### ISSUE TITLE
mobx-react-devtools not found

### ISSUE THREAD

**Issue by moimael** [philcockfield/storybook-host#2]
Hi,

I wanted to try your addon but when I add it to my project I get this error : 

```
ERROR in ./~/storybook-host/lib/components/ComponentHost.js
Module not found: Error: Cannot resolve module 'mobx-react-devtools' in /home/mlavault/Development/galaxy/node_modules/storybook-host/lib/components
 @ ./~/storybook-host/lib/components/ComponentHost.js 7:28-58
```

Because it is imported directly by your ComponentHost. Of course if I install the dependency manually I can launch storybook, but this shouldn't be needed.

Thanks !

**Comment by codeheroics** [comment]
@philcockfield Could you fix this, as this currently breaks the package, or would you accept a PR to either add mobx-react-devtools as a dependency (instead of a devDependency) or remove it?

**Comment by philcockfield** [comment]
Hey, sorry about this guys.  Fixed on version `1.0.7`
